### DISCUSSION TITLE
Can I receive Qdrant bounty payments in India via Algora as a sole proprietor?

### DISCUSSION THREAD

**Discussion by sohang3112** [qdrant/qdrant#2297] [Q&A]
Hi everyone 👋  I'm not sure if this is the right place to ask this question - please let me know if there's a better place to ask such questions. 🙂

I want to contribute to Qdrant’s Github repo and earn bounties, but the payments are made via Algora. According to [Algora’s documentation](https://docs.algora.io/bounties/payments#country-support), receiving international payments in India is limited to sole proprietors, limited liability partnerships, and companies. Can I set up payments with Algora as a sole proprietor to receive bounty payments?

**Comment by zodman** [comment]
you should ask to the algora.io people ... ask directly to twitter...
